### PR TITLE
Util: fix SystemUtil#getImplementationVersion for Java 9 using an alternative implementation

### PR DESCRIPTION
**Pull request by cryptokat**
This implementation stores commit version in an independent resource file.

### CODE REVIEW

**Comment by cryptokat** [comment]
`ConsoleDialogTest` is constantly falling on Travis-ci with Oracle Java for some reason which is affecting every pull request merging into `develop` branch. I am still trying to fix the test case in this pull request.

**Comment by orogvany** [comment]
Hm, well that was me that broke it :/

I'll take a look.

**Comment by cryptokat** [comment]
@orogvany 
The problem is reproducible locally with this command:
```
xvfb-run --server-args="-screen 0 1024x768x24" mvn -Dtest=ConsoleDialogTest test
```

File: src/test/java/org/semux/gui/dialog/ConsoleDialogTest.java [hunk 1]
```diff
@@ -43,6 +43,9 @@ public void testBasicUse() throws InterruptedException {
         DialogFixture console = window.dialog("Console").requireVisible();
         JTextComponentFixture consoleText = console.textBox("txtConsole");
 
+        // check txtInput
+        console.textBox("txtInput").click().requireFocused().requireEditable().requireVisible();
```
Comment (orogvany) [inline src/test/java/org/semux/gui/dialog/ConsoleDialogTest.java]: very strange.  glad it fixed it though

**Pull request merged**